src/main.js

```diff
@@ -684,7 +688,7 @@ class MiniGraphCard extends LitElement {
   }
 
   get visibleEntities() {
-    return this.config.entities.filter(entity => entity.show_graph !== false);
+    return this.config.entities.filter(entity => entity.enabeled && entity.show_graph !== false);
   }
```

Comment:
We can't really do it like this to decide what to render throughout the card since `show_graph` should only hide the entity from the "graph" part of the card, the state would still be visible e.t.c.

Reply:
why we can't? to be visible graph should be valid(enabled) and show_graph, this logic completely solves current problem

Reply:
`show_graph: false` does not mean the user want to hide the entity from the card completely, it means the entity should not render in the graph but a user might still want to see the state, extrema e.t.c.

So if we use `visibleEntities` to decide what state to render we won't render the state of an entity that has `show_graph: false` set even though we should. 
https://github.com/nurikk/mini-graph-card/blob/fix_missing_entity/src/main.js#L328-L329